clang/lib/AST/ExprConstant.cpp

```diff
@@ -7385,8 +7385,13 @@ class APValueToBufferConverter {
       for (size_t I = 0, E = CXXRD->getNumBases(); I != E; ++I) {
         const CXXBaseSpecifier &BS = CXXRD->bases_begin()[I];
         CXXRecordDecl *BaseDecl = BS.getType()->getAsCXXRecordDecl();
+        const APValue &Base = Val.getStructBase(I);
 
-        if (!visitRecord(Val.getStructBase(I), BS.getType(),
+        // Can happen in error cases.
+        if (!Base.isStruct())
```

Comment:
Curious what `Base` ends up being here?  Are we putting a NULL in here or something?

Reply:
Kinda, it's a `None` APValue.

Reply:
Ah, cool!  This for the clarification.